app/models/concerns/spotlight/solr_document/active_model_concern.rb

```diff
@@ -10,7 +10,7 @@ module Spotlight::SolrDocument::ActiveModelConcern
   module ClassMethods
 
     def primary_key
-      :id
+      self.unique_key
```

Comment:
I've got the following line in app/models/solr_document.rb in my app:
self.unique_key = 'pid'
Is there anything else that needs to be set?